src/app/plan/duck/operations.ts

```diff
@@ -0,0 +1,70 @@
+import { Creators as AlertCreators } from '../../common/duck/actions';
+import { Creators } from './actions';
+import { ClientFactory } from '../../../client/client_factory';
+import { IClusterClient } from '../../../client/client';
+import { MigResource, MigResourceKind } from '../../../client/resources';
+
+const migPlanFetchSuccess = Creators.migPlanFetchSuccess;
+const addPlanSuccess = Creators.addPlanSuccess;
+const addPlanFailure = Creators.addPlanFailure;
+const removePlanSuccess = Creators.removePlanSuccess;
+const removePlanFailure = Creators.removePlanFailure;
+
+const addPlan = migPlan => {
```

Comment:
Nice :+1: 

Still irks me that these have started to really all look the same. There will be some divergence in some places, like `AddCluster`, since the act of adding a cluster actually requires the creation of 3 distinct objects.

There's probably a nice way for us to clean this up, but it's a great example of something that is *not* on the critical path right now, so we should file an issue and circle back at another point in time. I'll file it.